Havtorn/Source/Editor/Windows/HierarchyWindow.cpp

```diff
@@ -40,15 +42,11 @@ namespace ImGui
 			Havtorn::I32 index = 0;
 			auto& entities = Scene->GetEntities();
 			
-			for (auto& entity : entities) 
+			for(Havtorn::U64 i = Havtorn::UDebugShapeSystem::MaxShapes; i < entities.size(); i++)
```

Comment:
why did this change?

Reply:
Removed the filtering by name (searching string). Debug shapes are added first, before other entities, so editor can loop starting from the last debug shape's index.